DataManagementSystem/Client/DataManager.py

```diff
@@ -1513,17 +1486,29 @@ def __filterTapeReplicas( self, replicaDict, diskOnly = False ):
     """
     Check a replica dictionary for disk replicas:
     If there is a disk replica, removetape replicas, else keep all
+    The input argument is modified
     """
-    for replicas in replicaDict['Successful'].values():
-      for se in replicas:
-        if diskOnly or self.__SEActive( se, access = 'DiskSE' ):
-          # There is one disk replica, remove tape replicas and exit loop
-          for se in replicas.keys():
-            if self.__SEActive( se, access = 'TapeSE' ):
-              replicas.pop( se )
-          break
-
-    return S_OK( replicaDict )
+    for lfn, replicas in replicaDict['Successful'].items():
```

Comment:
Please, just add a note here that `iteritems()` instead of `items()` won't work for this case (it would avoid brainless/bored changes in the future)